cmd/mt-split-metrics-by-ttl/main.go

```diff
@@ -68,23 +68,26 @@ func main() {
 	if err != nil {
 		panic(fmt.Sprintf("Failed to instantiate cassandra: %s", err))
 	}
-	tables := store.GetTableNames()
 
 	// create directory/link structure that we need to define the future table names
 	err = os.Mkdir(path.Join(tmpDir, storeConfig.Keyspace), 0700)
 	if err != nil {
 		panic(fmt.Sprintf("Failed to create directory: %s", err))
 	}
-	namedTableLinks := make([]string, len(tables))
-	for i, table := range tables {
-		namedTableLinks[i] = path.Join(tmpDir, storeConfig.Keyspace, table)
+	namedTableLinks := make([]string, len(store.TTLTables))
+	tableNames := make([]string, len(store.TTLTables))
+	i := 0
```

Comment:
not sure if missing something, i think this can still be initialized in `for` as it was before